examples/boltzmann_wealth_model/app.py

```diff
@@ -0,0 +1,127 @@
+import mesa
+
+from boltzmann_wealth_model.model import BoltzmannWealthModel
+
+import streamlit as st
+
+import time
+
+import pandas as pd
+
+import altair as alt
+
+import plotly.express as px
```

Comment:
Since `plotly` is no longer used, it should be removed from the import. It's a huge dependency.

Reply:
Sure